### DISCUSSION TITLE
Blur for rounded "Applications"

### DISCUSSION THREAD

**Discussion by kauanmocelin** [aunetx/blur-my-shell#627] [General]
Is there a way to solve this bug on rounded corners for gtk applications?

![Screenshot from 2024-07-28 23-29-20](https://github.com/user-attachments/assets/05b4f860-a219-49e7-9fff-a5af00e9829f)

**Comment by veudal** [comment]
A workaround that i personally found for myself is to set The Brightness to 1.00 under "Customize properties" in the applications tab under the "blur-my-shell" extension settings.
